app/scripts/lib/rpc-method-middleware/handlers/ethereum-chain-utils.js

```diff
@@ -176,9 +173,8 @@ export function validateAddEthereumChainParams(params) {
  * @param {Function} hooks.setActiveNetwork - The callback to change the current network for the origin.
  * @param {Function} hooks.endApprovalFlow - The optional callback to end the approval flow when approvalFlowId is provided.
  * @param {Function} hooks.getCaveat - The callback to get the CAIP-25 caveat for the origin.
- * @param {Function} hooks.requestPermissionApprovalForOrigin - The callback to prompt the user for permission approval.
- * @param {Function} hooks.updateCaveat - The callback to update the CAIP-25 caveat value.
- * @param {Function} hooks.grantPermissions - The callback to grant a CAIP-25 permission when one does not already exist.
+ * @param hooks.requestPermittedChainsPermissionForOrigin
+ * @param hooks.requestPermittedChainsPermissionIncrementalForOrigin
```

Comment:
Should we add a jsdoc descriptor for these hooks?

Reply:
https://github.com/MetaMask/metamask-extension/pull/29213/commits/8dae536ea3cbea94b4ba7945c02da1fa880218c9